docs/sphinx/conversion.rst

```diff
@@ -284,20 +284,19 @@ The unsafe example above, has been rewritten to use
     }
 
 Rather than managing the memory by hand, responsibility for this is
-delegated to a “smart” pointer, :cpp:class:`ome::compat::shared_ptr`.
-The memory is freed by the :cpp:class:`ome::compat::shared_ptr`
-destructor which is run at the end of the block scope, on explicit
-:c:type:`return`, or when cleaned up by exception stack unwinding.
+delegated to a “smart” pointer, :cpp:class:`std::shared_ptr`.  The
+memory is freed by the :cpp:class:`std::shared_ptr` destructor which
+is run at the end of the block scope, on explicit :c:type:`return`, or
+when cleaned up by exception stack unwinding.
 
 .. note::
 
-    :cpp:class:`ome::compat::shared_ptr` is either a
-    :cpp:class:`std::shared_ptr` or a :cpp:class:`boost::shared_ptr`,
-    depending upon whether C++11 features are avaiable or not,
-    respectively.
+    :cpp:class:`std::shared_ptr` is either a class:`std::shared_ptr`
+    :cpp:or a :cpp:class:`boost::shared_ptr`, depending upon whether
+    :cpp:C++11 features are avaiable or not, respectively.
```

Comment:
I've deleted this whole section since now C++11 is required this is all obsolete.